build.gradle

```
@@ -57,6 +59,12 @@ allprojects {
         parallel = true
         debug = false
     }
+
+    tasks.withType(KotlinCompile).all {
```

```suggestion
    tasks.withType(KotlinCompile).configureEach {
```

WDYT about configuring this task lazily? The impact here will be not noticeable, but I think it's a good hygiene to use more performant alternative.

---

This is now done: da917d92a406d669f657acc95f6d413dbee57c84